templates/repo/issue/view_content.tmpl

```diff
@@ -146,6 +146,15 @@
 								<div class="item text purple">
 									{{$.i18n.Tr "repo.pulls.has_merged"}}
 								</div>
+								{{if .IsPullBranchDeletable}}
+									<div class="ui divider"></div>
+									<div>
+										<form class="ui form" action="{{.DeleteBranchLink}}" method="post">
+											{{.CsrfTokenHtml}}
+											<button class="ui red button">Delete branch</button>
```

Comment:
Please make the displayable text translatable. Use `$.i18n.Tr` data evaluation structure for that purpose.
Here is an example:
`{{$.i18n.Tr "repo.pulls.has_merged"}}`